app/assets/stylesheets/questions.css

```
@@ -0,0 +1,10 @@
+.card {
+	max-width: 50rem;
+	height: 7rem
+}
+
+.card:hover {
+	transform: translateY(-4px);
+	border: 2px solid;
+	transition: 0.5s
+}
```

- `transition: 0.5s` は `.card` の方に入れないと，ホバー が外れた際にすぐに戻ってしまいます。
- 拡張子を`scss` に変更して，次のように書くとよいですね。

```suggestion
.card {
  max-width: 50rem;
  height: 7rem;
  transition: 0.5s;

  &:hover {
    transform: translateY(-4px);
    border: 2px solid;
  }
}
```